src/views/AdminSettings.vue

```diff
@@ -0,0 +1,105 @@
+  - GNU Affero General Public License for more details.
+  -
+  - You should have received a copy of the GNU Affero General Public License
+  - along with this program. If not, see <http://www.gnu.org/licenses/>.
+  -
+  -->
+
+<template>
+	<SettingsSection :title="t('notifications', 'Notifications defaults')">
+		<p class="settings-hint">
+			{{ t('notifications', 'Configure the default notification settings for new users') }}
+		</p>
+
+		<p>
+			<label for="notify_setting_batchtime" class="notification-frequency__label">
+				{{ t('notifications', 'Send email reminders about unhandled notifications after:') }}
+			</label>
+			<select id="notify_setting_batchtime"
+							v-model="config.setting_batchtime"
+							class="notification-frequency__select"
+							@change="updateSettings()">
+				<option v-for="option in batchtime_options" :key="option.value" :value="option.value">
+					{{ option.text }}
+				</option>
+			</select>
+		</p>
+
+		<CheckboxRadioSwitch :checked.sync="config.sound_notification"
+												 @update:checked="updateSettings">
+			{{ t('notifications', 'Play sound when a new notification arrives') }}
+		</CheckboxRadioSwitch>
+		<CheckboxRadioSwitch :checked.sync="config.sound_talk"
+												 @update:checked="updateSettings">
+			{{ t('notifications', 'Play sound when a call started (requires Nextcloud Talk)') }}
+		</CheckboxRadioSwitch>
+	</SettingsSection>
+</template>
+
+<script>
+import axios from '@nextcloud/axios'
+import {generateOcsUrl} from '@nextcloud/router'
+import {loadState} from '@nextcloud/initial-state'
+import {showSuccess, showError} from '@nextcloud/dialogs'
+import CheckboxRadioSwitch from '@nextcloud/vue/dist/Components/CheckboxRadioSwitch'
+import SettingsSection from '@nextcloud/vue/dist/Components/SettingsSection'
+
+const EmailFrequency = {
+	EMAIL_SEND_OFF: 0,
+	EMAIL_SEND_HOURLY: 1,
+	EMAIL_SEND_3HOURLY: 2,
+	EMAIL_SEND_DAILY: 3,
+	EMAIL_SEND_WEEKLY: 4,
+}
+
+export default {
+	name: 'AdminSettings',
+	components: {
+		CheckboxRadioSwitch,
+		SettingsSection,
+	},
+
+	data() {
+		return {
+			batchtime_options: [
+				{text: t('notifications', 'Never'), value: EmailFrequency.EMAIL_SEND_OFF},
+				{text: t('notifications', '1 hour'), value: EmailFrequency.EMAIL_SEND_HOURLY},
+				{text: t('notifications', '3 hours'), value: EmailFrequency.EMAIL_SEND_3HOURLY},
+				{text: t('notifications', '1 day'), value: EmailFrequency.EMAIL_SEND_DAILY},
+				{text: t('notifications', '1 week'), value: EmailFrequency.EMAIL_SEND_WEEKLY},
+			],
+			config: loadState('notifications', 'config'),
+		}
+	},
+
+	methods: {
+		async updateSettings() {
+			try {
+				const form = new FormData()
+				form.append('batchSetting', this.config.setting_batchtime)
+				form.append('soundNotification', this.config.sound_notification ? 'yes' : 'no')
+				form.append('soundTalk', this.config.sound_talk ? 'yes' : 'no')
+				await axios.post(generateOcsUrl('apps/notifications/api/v2/settings/admin'), form)
```

Comment:
Instead of adding our own API call + endpoint here we could use
```
OCP.AppConfig.setValue()
```

But then it's not possible to have a sub admin permission for this, although I'm not convinced that would be necessary at all